### PR TITLE
track_growth: take lat/lon as query parameters

### PR DESCRIPTION
**Pull request by markspolakovs**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
 app.mount("/ui", StaticFiles(directory="app/static", html=True), name="static")
 
 # Set up PostgreSQL
-config = load_database_config("config.ini")
```
Comment (BabarKhanDev) [inline app/main.py:41]: This change will break the app if we want to run it locally rather than in docker

**Review by BabarKhanDev — CHANGES_REQUESTED** [review]
Heya, happy with the changes to track growth
for the config maybe we want to change load_database_config() itself to look more like this:
```
def load_database_config(filename: str = "../config.ini"):
    filename = os.getenv('CONFIG_FILE', filename)
```
that way we can pass in an arg and it wont matter if we have an environment variable set

**Comment by markspolakovs** [comment]
I’ll revert the config change and possibly PR that separately.

**Review by BabarKhanDev — APPROVED** [review]
Lgtm

**Pull request merged**